src/serac/physics/state/state_manager.cpp

```diff
@@ -53,13 +53,28 @@ double StateManager::newDataCollection(const std::string& name, const std::optio
 
     // Functional needs the nodal grid function and neighbor data in the mesh
 
+    // Determine if the existing nodal grid function is discontinuous. This
+    // indicates that the mesh is periodic and the new nodal grid function must also
+    // be discontinuous.
+    bool is_discontinuous = false;
```

Comment:
Can these two duplicate sections be unified into a function w/o too much effort?